packages/gatsby-plugin-manifest/src/pluginOptionsSchema.js

```diff
@@ -0,0 +1,236 @@
+    src: Joi.string()
+      .required()
+      .description(
+        `The path to the image file. If src is a relative URL, the base URL will be the URL of the manifest.`
+      ),
+    type: Joi.string().description(
+      `A hint as to the media type of the image. The purpose of this member is to allow a user agent to quickly ignore images with media types it does not support.`
+    ),
+    purpose: Joi.string()
+      .optional()
+      .description(
+        `Defines the purpose of the image, for example if the image is intended to serve some special purpose in the context of the host OS.`
+      ),
+  })
+
+  const ShortcutItem = Joi.object().keys({
+    name: Joi.string()
+      .required()
+      .description(
+        `The name member of a ShortcutItem is a string that represents the name of the shortcut as it is usually displayed to the user in a context menu. `
+      ),
+    short_name: Joi.string()
+      .optional()
+      .description(
+        `The short_name member of a ShortcutItem is a string that represents a short version of the name of the shortcut. `
+      ),
+    description: Joi.string()
+      .optional()
+      .description(
+        `The description member of a ShortcutItem is a string that allows the developer to describe the purpose of the shortcut. `
+      ),
+    url: Joi.string()
+      .required()
+      .description(
+        `The url member of a ShortcutItem is a URL within scope of a processed manifest that opens when the associated shortcut is activated. `
+      ),
+    icons: Joi.array()
+      .optional()
+      .items(ManifestImageResource)
+      .description(
+        `The icons member of an ShortcutItem member serve as iconic representations of the shortcut in various contexts. `
+      ),
+  })
+
+  const ExternalApplicationResource = Joi.object()
+    .keys({
+      platform: platform.required(),
+      url: Joi.string()
+        .uri()
+        .required()
+        .description(`The URL at which the application can be found.`),
+      id: Joi.string()
+        .required()
+        .description(
+          `The ID used to represent the application on the specified platform.`
+        ),
+      min_version: Joi.string()
+        .optional()
+        .description(
+          `The minimum version of the application that is considered related to this web app.`
+        ),
+      fingerprints: Joi.array()
+        .optional()
+        .items(FingerPrint)
+        .description(
+          `Each Fingerprints represents a set of cryptographic fingerprints used for verifying the application.`
+        ),
+    })
+    .or("url", "id")
+
+  const WebAppManifest = Joi.object().keys({
+    background_color: Joi.string()
+      .optional()
+      .description(
+        `The background_color member defines a placeholder background color for the application page to display before its stylesheet is loaded.`
+      ),
+    categories: Joi.array()
```

Comment:
The spec recommends

> Manifest authors are encouraged to use lower-case.

So could we consider using `.lowercase()` here?

Reply:
I thought about this too. What I didn't want is it to throw an error if the user didn't put in all lowercase. But conversion should be enabled so it's probably fine.

Reply:
agreed, will resolve this comment